scania-digital-design-system/sdds
# fix(table): inline json dummy data

**Describe pull-request**  
A bug report in the support channel revealed that when using the @scania/tegel package in a stencil.js project threw an error that was traced back to the external dummy-json file being used in the table-body.tsx-file. Proposal is to inline it instead. This has been tried and verified in a local stencil.js project. 

**Solving issue**  
Fixes: [DTS-1459](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?selectedIssue=DTS-1459)

**How to test**  
_Add description how to test if possible_
1. Go to Storybook
2. Check in Components -> Table components, and see that the data is displayed.

**Suggested test steps**



[DTS-1459]: https://tegel.atlassian.net/browse/DTS-1459?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
